pkg/storage/swift/swift.go

```diff
@@ -51,6 +51,16 @@ func replaceEmpty(a string, b string) string {
 	return a
 }
 
+// IsSwiftEnabled checks if Swift service is available for OpenStack platform
+func IsSwiftEnabled(listers *regopclient.Listers) bool {
+	driver := NewDriver(&imageregistryv1.ImageRegistryConfigStorageSwift{}, listers)
+	_, err := driver.getSwiftClient()
```

Comment:
Not every error returned by this function indicates that the Swift is disabled, don't you think?

Reply:
agree with you. I'll update the patch soon

Reply:
done